journalbeat/tests/system/test_base.py

```diff
@@ -143,19 +121,7 @@ def test_read_events_with_existing_registry(self):
         )
         journalbeat_proc = self.start_beat()
 
-        required_log_snippets = [
-            # journalbeat can be started
-            "journalbeat is running",
-            # journalbeat can seek to the position defined in the cursor
-            "Seeked to position defined in cursor",
-            # message can be read from test journal
-            "please report the conditions when this event happened to",
-            # only one event is read and published
-            'journalbeat successfully published events\t{"event.count": 1}',
-        ]
-        for snippet in required_log_snippets:
-            self.wait_until(lambda: self.log_contains(snippet),
-                            name="Line in '{}' Journalbeat log".format(snippet))
+        self.wait_until(lambda: self.output_has(lines=9))
```

Comment:
Same here.

Once more thing: how likely is this output to not contain `9` lines?